src/Question/Show/Answer/View.fs

```diff
@@ -1,25 +1,26 @@
 module Question.Show.Answer.View
 
 open Types
-open Fable.Helpers.React
-open Fable.Helpers.React.Props
+open Fable.React
+open Fable.React.Props
 open Fulma
-open Fulma.FontAwesome
+open Fable.FontAwesome
+open Fable.FontAwesome.Free
 open System
 
 let private voteArea score =
     let icon =
         if score > 0 then
-            Fa.I.ThumbsOUp
+            Fa.Regular.ThumbsUp
         else
-            Fa.I.ThumbsODown
+            Fa.Regular.ThumbsDown
 
     let iterationCount =
         Math.Min(5, Math.Abs score - 1)
 
     span [ ]
         [ for i = 0 to iterationCount do
-            yield Icon.faIcon [ ] [ Fa.icon icon ] ]
+            yield Fa.i [icon] [] ]
```

Comment:
Should be something like:

```fs
Icon.icon [ ]
    [ Fa.i [ icon ]
        [ ] ]
```

Reply:
Then it may be a good idea to update the documentation, I did following these samples: https://fulma.github.io/Fulma/#fable-fontawesome/usage

Reply:
I changed this and the github icons, thank you! I've rebased the first commit so we can still focus the second on the Fable.Core 3 update :+1: